specification/components/schemas/eRS-Appointment.yaml

```diff
@@ -107,6 +107,24 @@ properties:
                     type: string
                     nullable: true
                     example: 'Provider Converted Advice and Guidance, Admin to Refer'
+              - type: object
```

Comment:
I can't comment on the line but can we remove the top level description for Appointment? Seems this is just coped from FHIR, don't think it adds anything.